config/index.js

```diff
@@ -334,6 +334,32 @@ var conf = convict({
       env: 'SUPPORTED_LANGS'
     }
   },
+  redis: {
+    enabled : {
+      default: true,
+      doc: 'Enable redis cache',
+      format: Boolean,
+      env: 'USE_REDIS'
+    },
+    host: {
+      default: '127.0.0.1',
+      env: 'REDIS_HOST',
+      format: String,
+      doc: 'Url for redis host',
+    },
+    port: {
+      default: 6379,
+      env: 'REDIS_PORT',
+      format: 'port',
+      doc: 'port for redis server'
+    },
+    sessionsKeyPrefix: {
+      default: 'fxa-auth-session',
+      env: 'REDIS_KEY_PREFIX',
+      format: String,
+      doc: 'which key prefix to redis cache'
```

Comment:
`which key prefix to redis cache` needs to change to reflect that this is for the session store , not really a `cache` either